cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestTimeSeriesCategoryEndpointInput.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2021 Hydrologic Engineering Center
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package mil.army.usace.hec.cwms.radar.client.controllers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import org.junit.jupiter.api.Test;
+
+class TestTimeSeriesCategoryEndpointInput {
+
+    @Test
+    void testTimeSeriesCategoryEndpointInputGetAll() {
+        TimeSeriesCategoryEndpointInput timeSeriesCategoryEndpointInput = new TimeSeriesCategoryEndpointInput();
+        assertFalse(timeSeriesCategoryEndpointInput.getCategoryId().isPresent());
+    }
+
+    @Test
+    void testTimeSeriesCategoryEndpointInputCategoryId() {
+        TimeSeriesCategoryEndpointInput timeSeriesCategoryEndpointInput = new TimeSeriesCategoryEndpointInput("category-id");
+        assertEquals("category-id", timeSeriesCategoryEndpointInput.getCategoryId().get());
```

Comment:
no test on office id

Reply:
Need to wait for Bryson's mock work to test that

Reply:
https://jira.hecdev.net/browse/CWMS-687